boa3/analyser/typeanalyser.py

```diff
@@ -1196,15 +1193,51 @@ def validate_callable_arguments(self, call: ast.Call, callable_target: Callable)
 
         ignore_first_argument = int(callable_target.has_cls_or_self)  # 1 if True, 0 otherwise
         len_call_args = len(call.args)
+        len_call_keywords = len(call.keywords)
         callable_required_args = len(callable_target.args_without_default) - ignore_first_argument
 
-        if len_call_args > len(callable_target.args):
-            unexpected_arg = call.args[len(callable_target.args) + ignore_first_argument]
+        # verifies if a non-default arg is being called as a keyword
+        necessary_kwars = list(callable_target.args_without_default.keys())[len_call_args:callable_required_args]
```

Comment:
```suggestion
        necessary_kwargs = list(callable_target.args_without_default.keys())[len_call_args:callable_required_args]
```
I wouldn't reprove the pr just because of this, but since we have other issues here
just rename the variable